docs/updating/6_finish_the_update.md

```diff
@@ -49,7 +49,7 @@ git merge <branch_name>
     To ensure that no users have unsupported, insecure password hashes, run the following command:
     
     ``` bash
-    php bin/console ezplatform:user:validate-password-hashes
+    php bin/console ibexa:user:validate-password-hashes
```

Comment:
If I understand correctly this page is meant for users using all possible versions:
```
# In v1 and v2:
php bin/console assetic:dump -e prod
# In v2 and v3:
yarn install
yarn encore prod
```

So maybe we should mention that for version 3.2 you should run:
`php bin/console ezplatform:user:validate-password-hashes`
For 3.3:
`php bin/console ibexa:user:validate-password-hashes`
And for v1 and v2 it's not available.